tools/devops/provision-brew-packages.csx

```diff
@@ -1,9 +1,11 @@
-BrewPackages ("cmake", 
+BrewPackages (
+    "cmake", 
     "autoconf",
     "automake",
     "libtool",
     "p7zip",
     "python",
     "libmagic", 
-    "msitools",
-    "wget");
+    "msitools"
```

Comment:
no `,` ?

Reply:
good catch, it is working because I disabled the step